example/pyproject.toml

```diff
@@ -2,15 +2,15 @@
 name = "connect-python-example"
 version = "0.1.0"
 
-dependencies = ["connect-python", "flask", "protobuf", "starlette"]
-
-[tool.ruff]
-extend = "../pyproject.toml"
+dependencies = ["connect-python", "flask>=3.1.2", "protobuf>=5.28", "starlette>=0.48.0"]
 
 [build-system]
 requires = ["uv_build>=0.7.21,<0.8.0"]
 build-backend = "uv_build"
 
+[tool.ruff]
+extend = "../pyproject.toml"
```

Comment:
Thanks always forget this

Reply:
it just moved down in the diff, don't worry, you didn't forget :).

my linter was just complaining about the toml ordering of keys; I'll leave this change but not bother with it in the future.